Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackFormViewModel.swift

```diff
@@ -19,6 +19,11 @@ class SentryUserFeedbackFormViewModel: NSObject {
     unowned let controller: SentryUserFeedbackFormController
     weak var delegate: SentryUserFeedbackFormViewModelDelegate?
     
+    /// Checks to make sure the app provides the necessary Info plist key to request authorization. If the key is not present, trying to interact with certain Photos APIs will crash the app.
+    var canRequestAuthorizationToAttachPhotos = {
+        Bundle.main.infoDictionary?["NSPhotoLibraryUsageDescription"] != nil
+    }()
```

Comment:
`m`: Wouldn't it make sense to check that once the user configures `formConfig.enableScreenshot = true`? Then the user would immediately see the error message. I would even change the log level to error. Now, we wait until we try to attach a photo.

Reply:
This will prevent the button from even appearing in the form for now, but I will also make a check at the point the configuration is set in the call to SentrySDK.startWithOptions to emit an earlier log.

Reply:
This will now also log a warning at the time the integration is installed:

![image](https://github.com/user-attachments/assets/cdbd95cd-87d8-4bcf-a6da-857a598e8938)

Reply:
shoot, forgot to push the commit before merging this. i pushed it to https://github.com/getsentry/sentry-cocoa/pull/4698/commits/d6aecd5952d318160021cf7bc18e95cbd1f99075